### PR TITLE
waydroid: 1.4.1 -> 1.4.2

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Diff: https://github.com/waydroid/waydroid/compare/1.4.1...1.4.2

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/os-specific/linux/waydroid/default.nix [hunk 1]
```diff
@@ -61,18 +69,15 @@ python3Packages.buildPythonApplication rec {
...
-      python3Packages.pygobject3
-      python3Packages.pyclip
+    ] ++ propagatedBuildInputs ++ [
       gawk
       kmod
       lxc
       util-linux
-      xclip
-    ]}"
+      wl-clipboard
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/waydroid/default.nix:79]: This fixes copy to clipboard in the default configuration

File: pkgs/os-specific/linux/waydroid/default.nix [hunk 2]
```diff
@@ -51,7 +51,7 @@ python3Packages.buildPythonApplication rec {
   dontWrapGApps = true;
 
   installPhase = ''
-    make install PREFIX=$out USE_SYSTEMD=0
+    make install DESTDIR=$out PREFIX= USE_SYSTEMD=0
```
Comment (CajuM) [inline pkgs/os-specific/linux/waydroid/default.nix:54]: Are you sure this is correct? Typically DESTDIR doesn't modify the contents of a package to indicate its location in the filesystem. While PREFIX is supposed to set its absolute location, at times in the package's files.

File: pkgs/os-specific/linux/waydroid/default.nix (hunk 1 above)
Comment (CajuM) [reply pkgs/os-specific/linux/waydroid/default.nix:71]: Apparently XWayland gave me a false sense of security...  Clipboard support now works without issue on Weston.

File: pkgs/os-specific/linux/waydroid/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/os-specific/linux/waydroid/default.nix:54]: Yes, PREFIX defaults to /usr which we don't want.

**Review by mkg20001 — APPROVED** [review]

File: pkgs/os-specific/linux/waydroid/default.nix (hunk 2 above)
Comment (NickCao) [reply pkgs/os-specific/linux/waydroid/default.nix:54]: > Yes, PREFIX defaults to /usr which we don't want.

That's why we set it to $out, see also https://github.com/NixOS/nixpkgs/issues/65718

Comment (SuperSandro2000) [reply pkgs/os-specific/linux/waydroid/default.nix]: ammended the fix

**Review by mkg20001 — APPROVED** [review]

File: pkgs/os-specific/linux/waydroid/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/os-specific/linux/waydroid/default.nix]: Can someone help me with #1218 ?

**Pull request merged**